.gitlab-ci.yml

```diff
@@ -1,18 +1,26 @@
 # SPDX-License-Identifier: GPL-2.0+
```

Comment:
I don't know if changing `.gitlab-ci.yml` can hurt anything. But maybe this commit is not exactly needed also.

Reply:
it's not even used in the mainline U-Boot now, so yeah it can be just dropped